### DISCUSSION TITLE
What does "mu" stand for?

### DISCUSSION THREAD

**Discussion by DavidBuchanan314** [unicorn-engine/unicorn#1979] [Q&A] [answered]
In a lot of the sample code, e.g. https://github.com/unicorn-engine/unicorn/blob/master/bindings/python/sample_x86.py, instances of the `Uc` class are named `mu`. But in other places, they're named `uc`. Presumably uc is short for unicorn, but what is mu, and why is it used in some places and not in others?

Obviously this isn't a very important question, I'm just curious. (and wondering what the "right" thing to call it in my own code would be)

**Comment by gerph** [comment, marked as answer]
I believe this is just a play on the pronunciation of m - u. Which in English sounds em-you, for EMUlator.
